src/bin/common/solving_loop.ml

```diff
@@ -153,6 +153,23 @@ let main () =
     O.get_sat_solver (),
     (module SatCont.Make(TH) : Sat_solver_sig.S)
   in
+
+  let print_model ppf (Model ((module SAT), env)) =
+    let ur = SAT.get_unknown_reason env in
+    Printer.print_fmt (Options.Output.get_fmt_diagnostic ())
+      "@[<v 0>; Returned unknown reason = %a@]"
+      Sat_solver_sig.pp_unknown_reason_opt ur;
+    match SAT.get_model env with
+    | None ->
+      Printer.print_fmt (Options.Output.get_fmt_diagnostic ())
+        "@[<v 0>It seems that no model has been computed so \
+         far. You may need to change your model generation strategy \
+         or to increase your timeouts."
```

Comment:
Why change the error message?

Reply:
I didn't change it but I print the timeout reason just before. Now, if we don't print always the `unknown reason` while outputting the model, it makes sense to keep print it if we fail to produce a model here.